File: Java_Programmers/1단계/src/핸드폰번호가리기.java
public class 핸드폰번호가리기 {
    public String solution(String phone_number) {
        String answer = "";
        for (int i = 0; i < phone_number.length() - 4; i++) {
            answer += "*";
        }
        answer += phone_number.substring(phone_number.length() - 4);
        return answer;
    }
    public String solution2(String phone_number) {
        char[] ch = phone_number.toCharArray();
        for(int i = 0; i < ch.length - 4; i ++){
            ch[i] = '*';
        }
        return String.valueOf(ch);
    }

    public static void main(String[] args) {
        핸드폰번호가리기 sol = new 핸드폰번호가리기();
        String phone_number = "01033334444";
        System.out.println(sol.solution(phone_number));
        System.out.println(sol.solution2(phone_number));
    }
}

File: python/python_Programmers/1단계/핸드폰 번호 가리기.py
def solution(phone_number):
    answer = ''

    cnt = phone_number[:-4]
    phone_number = phone_number.replace(cnt, '*' * len(cnt))
    return phone_number

print(solution("01033334444"))

# 모범 풀이
def hide_numbers(s):
    return "*"*(len(s)-4) + s[-4:]

# 아래는 테스트로 출력해 보기 위한 코드입니다.
print("결과 : " + hide_numbers('01033334444'));